### ISSUE TITLE
Version number reported to raygun in the package version

### ISSUE THREAD

**Issue by RobinClowers** [Cobenian/raygun#7]
When are reported from our elixir app, the version number is the raygun package version, not our application's version.

**Comment by brweber2** [comment]
This one is a bit more tricky.  This project is a library so it doesn't have access to the configuration of the actual project that includes this library and we can't really make any assumptions about the project (even that it is a mix project). We could potentially open this up and allow the version to be passed in (via the plug config for example...).  What do you think about that as a proposed solution?  Would that work for you?

**Comment by RobinClowers** [comment]
Yeah, I think specifying it via the plug config would be fine.

**Comment by brweber2** [comment]
Before I make any code changes, have you tried adding this to your project's config (obviously with your name/version):

```
config :raygun,
    client_name: "MyApp",
    client_version: "2.3.4"
```

**Comment by brweber2** [comment]
It looks like the "version" and "client.version" are transposed in the format.ex.... I'm going to give this a little thought to see if there is something better I can do for handling application and library info.

**Comment by RobinClowers** [comment]
No, it seems we overlooked the bit in the readme about being able to specify the version. It would be great if it could be automatic though.

**Comment by brweber2** [comment]
OK, I've switch the two version values around.  Release as 0.1.5.  I'm going to leave this issue open for a little bit as I re-think my approach, but hopefully your particular issue is addressed in 0.1.5.

**Comment by RobinClowers** [comment]
Great, I'll test it out tomorrow and let you know.

**Comment by brweber2** [comment]
OK, I think I figured it out.  If you use Raygun.Plug it will auto-detect the app's version for you.  This was released in 0.1.6.

**Comment by aaronjensen** [comment]
Speaking for @RobinClowers, who is on my team, I can confirm that we are seeing version numbers now, so this can be closed out, I think.
